rs/nervous_system/canisters/src/registry.rs

```diff
@@ -133,6 +140,17 @@ impl FakeRegistry {
             .push(response);
     }
 
+    /// Encodes a prost message at latest_version + 1, and updates latest_version.
+    pub fn encode_value<T: prost::Message>(&self, key: impl AsRef<str>, value: Option<T>) {
+        self.encode_value_at_version(
+            key,
+            self.version.load(std::sync::atomic::Ordering::SeqCst) + 1,
```

Comment:
Could this overflow? Could we make it a `checked_add`?

If is enforced to be test-only, please disregard.